packages/validation/backend/sirius-components-collaborative-validation/src/test/java/org/eclipse/sirius/components/collaborative/validation/ValidationEventProcessorTests.java

```diff
@@ -0,0 +1,85 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.components.collaborative.validation;
+
+import java.util.List;
+import java.util.UUID;
+import java.util.function.Predicate;
+
+import org.eclipse.sirius.components.collaborative.api.IRepresentationRefreshPolicyRegistry;
+import org.eclipse.sirius.components.collaborative.representations.SubscriptionManager;
+import org.eclipse.sirius.components.collaborative.validation.dto.ValidationEventInput;
+import org.eclipse.sirius.components.collaborative.validation.dto.ValidationRefreshedEventPayload;
+import org.eclipse.sirius.components.core.api.IEditingContext;
+import org.eclipse.sirius.components.core.api.IInput;
+import org.eclipse.sirius.components.core.api.IObjectService;
+import org.eclipse.sirius.components.core.api.IPayload;
+import org.eclipse.sirius.components.validation.Validation;
+import org.eclipse.sirius.components.validation.description.ValidationDescription;
+
+import io.micrometer.core.instrument.simple.SimpleMeterRegistry;
+import reactor.test.StepVerifier;
+
+import org.junit.jupiter.api.Test;
+
+/**
+ * Unit tests of the ValidationEventProcessor.
+ *
+ * @author mcharfadi
+ */
+public class ValidationEventProcessorTests {
```

Comment:
Don't add more unit tests, contribute to the integration tests if needed, there are already test suite for the validation

Reply:
Done